HPXMLtoOpenStudio/measure.rb

```diff
@@ -2283,8 +2285,9 @@ def add_airflow(runner, model, weather, spaces, airloop_map)
 
     # Create HVAC availability sensor
     hvac_availability_sensor = nil
+    @hvac_unavailable_periods = @hvac_unavailable_periods.flatten
     if not @hvac_unavailable_periods.empty?
-      avail_sch = ScheduleConstant.new(model, SchedulesFile::Columns[:HVAC].name, 1.0, EPlus::ScheduleTypeLimitsFraction, unavailable_periods: @hvac_unavailable_periods)
+      avail_sch = ScheduleConstant.new(model, 'hvac', 1.0, EPlus::ScheduleTypeLimitsFraction, unavailable_periods: @hvac_unavailable_periods)
```

Comment:
What is this about?

Reply:
The `HVAC` key was replaced by `SpaceHeating` and `SpaceCooling`. But we still want to create a constant schedule (1s) with unavailable period rules (0s) that we can put an EMS sensor on.

Reply:
I see. I guess it's fine. I would have thought the name should be more descriptive, but I guess it was simply "hvac" before?

Reply:
Yes, it was "hvac" before. I can update it to be a bit more descriptive.